### PR TITLE
Surface Worker Service project template

### LINKED ISSUE #11791: Surface 3.0 Windows service project template in WS topic
[.NET Core Workers as Windows Services](https://devblogs.microsoft.com/aspnet/net-core-workers-as-windows-services/)

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #11791 

* Gets it into the hosted services and WS topics.
* I'll surface the hosted services VS4Mac instructions at 3.0 RTM. https://github.com/aspnet/AspNetCore.Docs/issues/12664

btw - In the WS topic, I tried a version of this where the IDE instructions are under item 1 in the two-item list. It renders with a full-width horizontal rule under the list item that I'm not happy about. Therefore, I list the two steps and then show the IDE instructions. We can play with it if you want, but I don't think it's worth 😅 *sweat'in* 😅 ... got other stuff that *I gots to get to*. :smile:

### CODE REVIEW

File: aspnetcore/fundamentals/host/hosted-services.md [hunk 1]
```diff
@@ -25,6 +25,32 @@ The sample app is provided in two versions:
 * Web Host &ndash; Web Host is useful for hosting web apps. The example code shown in this topic is from Web Host version of the sample. For more information, see the [Web Host](xref:fundamentals/host/web-host) topic.
 * Generic Host &ndash; Generic Host is new in ASP.NET Core 2.1. For more information, see the [Generic Host](xref:fundamentals/host/generic-host) topic.
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Worker Service template
+
+The ASP.NET Core Worker Service template provides a starting point for writing long running service apps in ASP.NET Core. To use the template as a basis for a hosted services app:
```
Comment (tdykstra) [inline aspnetcore/fundamentals/host/hosted-services.md]:
```suggestion
The ASP.NET Core Worker Service template provides a starting point for writing long running service apps. To use the template as a basis for a hosted services app:
```

File: aspnetcore/host-and-deploy/windows-service.md [hunk 2]
```diff
@@ -21,6 +21,35 @@ An ASP.NET Core app can be hosted on Windows as a [Windows Service](/dotnet/fram
 * [ASP.NET Core SDK 2.1 or later](https://dotnet.microsoft.com/download)
 * [PowerShell 6.2 or later](https://github.com/PowerShell/PowerShell)
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Worker Service template
+
+The ASP.NET Core Worker Service template provides a starting point for writing long running service apps in ASP.NET Core. To use the template as a basis for a Windows Service app:
```
Comment (tdykstra) [inline aspnetcore/host-and-deploy/windows-service.md]:
```suggestion
The ASP.NET Core Worker Service template provides a starting point for writing long running service apps. To use the template as a basis for a Windows Service app:
```

File: aspnetcore/host-and-deploy/windows-service.md [hunk 3]
```diff
@@ -21,6 +21,35 @@ An ASP.NET Core app can be hosted on Windows as a [Windows Service](/dotnet/fram
 * [ASP.NET Core SDK 2.1 or later](https://dotnet.microsoft.com/download)
 * [PowerShell 6.2 or later](https://github.com/PowerShell/PowerShell)
 
+::: moniker range=">= aspnetcore-3.0"
+
+## Worker Service template
```
Comment (Rick-Anderson) [inline aspnetcore/host-and-deploy/windows-service.md]: @guardrex  if this is duplicated content, should I move it to an include?

Comment (guardrex) [reply aspnetcore/host-and-deploy/windows-service.md]: IMO, **_not just yet_** ... not until I get to https://github.com/aspnet/AspNetCore.Docs/issues/12841. IIRC, the light coverage on this template was provided just to surface its existence at the time. I have a feeling that that these two topics (background tasks and WS) aren't going to say the same thing about the use of the template, so an INCLUDE might not work out. Not sure ... I need to get to that issue and analyze the coverage and see what's what.

**Comment by Rick-Anderson** [comment]
We should put the info in one place, be it an include or a link to another topic.

**Pull request merged**